query.go

```diff
@@ -995,6 +1081,16 @@ func NewNumericRangeInclusiveQuery(min, max float64, minInclusive, maxInclusive
 	}
 }
 
+// Min returns tehe numeric range lower bound
+func (q *NumericRangeQuery) Min() float64 {
```

Comment:
OK, I noticed one last thing, these (Min/Max) should return `(float64, bool)` with `inclusiveMin` and `inclusiveMax` respectivley.  Sorry, I didn't notice that the first time.  If we add those I think it's ready to merge.

Reply:
Updated and fixed teh typo.